pkg/sfu/buffer/buffer.go

```diff
@@ -505,9 +505,10 @@ func (b *Buffer) Close() error {
 	b.closeOnce.Do(func() {
 		b.closed.Store(true)
 
-		b.RLock()
+		b.Lock()
```

Comment:
rlock and lock are mutually exclusive so this is probably ok either way

Reply:
yeah, will change it back.